docker/conf/homeserver.yaml

```diff
@@ -2,7 +2,7 @@
 
 ## TLS ##
 
-{% if not SYNAPSE_NO_TLS %}
+{% if not SYNAPSE_NO_TLS == '1' %}
```

Comment:
this would normally be spelt `if SYNAPSE_NO_TLS != '1'`.

In any case I'm concerned that this will give unexpected behaviour for anyone running with, for example, `SYNAPSE_NO_TLS=TRUE`. You may need to add some sort of string-to-bool conversion function to `start.py`.

I'd also suggest putting this in a separate PR, so that the trivial changes to the readme don't get held up by slightly more complicated changes here. (Or at least, fix the title of the PR.)

Reply:
OK. I looked up how to have multiple PR's open. I will reorganize.

I'll also rethink the attempted #4663 fix.

Reply:
Well, I can't figure out how to have two PR's going at the same time when hoping to merge to the same remote branch and keep the commits separated. For now, I created a draft PR that includes my fix to #4663: https://github.com/matrix-org/synapse/pull/5005